### DISCUSSION TITLE
Fire Event Hook for manual sort

### DISCUSSION THREAD

**Discussion by ivoamaro** [directus/directus#17644] [Feature Requests]
### Summary

I have a collection where I can create multiple items. 
In the collection I enable manual sorting where I can change the order of the items.

<img width="843" alt="Captura de ecrã 2023-02-28, às 10 15 06" src="https://user-images.githubusercontent.com/33934913/221824371-bd4fce18-b1cb-4394-aea8-f971140b92b1.png">

I would like to fire and event hook whenever I change the order. I thought this would trigger the update event, but nothing happens.

### Motivation

The idea behind this would be to trigger a webhook to rebuild a website. We could add a sort trigger or even make the update trigger fire when we change the items order

<img width="820" alt="Captura de ecrã 2023-02-28, às 10 21 08" src="https://user-images.githubusercontent.com/33934913/221825746-e098fd86-af5a-4477-80cc-5b3b6601d0ab.png">

### Detailed Design

-

### Requirements List

-

### Drawbacks

-

### Alternatives

-

### Adoption Strategy

-

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
